x/validator/keeper/msg_server_create_validator.go

```diff
@@ -21,10 +20,10 @@ func (k msgServer) CreateValidator(goCtx context.Context, msg *types.MsgCreateVa
 	}
 
 	// check if sender has enough rights to create a validator node
-	if !k.dclauthKeeper.HasRole(ctx, sdk.AccAddress(valAddr), dclauthtypes.NodeAdmin) {
+	if !k.dclauthKeeper.HasRole(ctx, sdk.AccAddress(valAddr), types.EnableDisableValidatorRole) {
```

Comment:
We require this role for each of the following 3 operations: `CreateValidator`, `DisableValidator` and `EnableValidator`, i.e. for any node control operation not based on voting process. The role `NodeAdmin` tells the same by its own name. So `EnableDisableValidatorRole` (or renamed to `CreateDisableEnableValidatorRole`) seems to be a redundant constant. Let's just remove this constant and use `NodeAdmin` directly.

Generally, we use constants referencing roles only for operations based on voting process. Now all of them reference `Trustee` role because now any voting is performed by trustees. To be able in future to change the voting roles in different modules independently, we use constants for voting operations.